crates/bdk/src/wallet/mod.rs

```diff
@@ -54,7 +54,6 @@ pub(crate) mod utils;
 pub mod error;
 pub use utils::IsDust;
 
-#[allow(deprecated)]
```

Comment:
❓ I'm actually not sure why this was here before

Reply:
Me neither, but always glad to nuke it out of the codebase.
More strict lints checks are always welcomed...

Reply:
OOOOOO sorry. Yes, this was here because we were using the deprecated `rust-miniscript`'s `max_satisfaction_weight`.

That's the whole point of the PR 😂